ionoscloud/resource_server_test.go

```diff
@@ -769,3 +827,128 @@ resource ` + ServerResource + ` ` + ServerTestResource + ` {
 	  }
     }
 }`
+
+const testAccCheckDataSourceServerWithLabels = testAccCheckServerCreationWithLabels + `
+data ` + ServerResource + ` ` + ServerDataSourceById + ` {
+  datacenter_id = ` + DatacenterResource + `.` + DatacenterTestResource + `.id
+  id			= ` + ServerResource + `.` + ServerTestResource + `.id
+}
+`
+
+const (
+	testAccCheckServerUpdateLabels = `
+resource ` + DatacenterResource + ` ` + DatacenterTestResource + ` {
+	name       = "server-test"
+	location = "us/las"
+}
+resource "ionoscloud_ipblock" "webserver_ipblock" {
+  location = ` + DatacenterResource + `.` + DatacenterTestResource + `.location
+  size = 4
+  name = "webserver_ipblock"
+}
+resource ` + LanResource + ` ` + LanTestResource + ` {
+  datacenter_id = ` + DatacenterResource + `.` + DatacenterTestResource + `.id
+  public = true
+  name = "public"
+}
+resource ` + ServerResource + ` ` + ServerTestResource + ` {
+  name = "` + ServerTestResource + `"
+  datacenter_id = ` + DatacenterResource + `.` + DatacenterTestResource + `.id
+  cores = 1
+  ram = 1024
+  availability_zone = "ZONE_1"
+  cpu_family = "AMD_OPTERON"
+  image_name ="ubuntu:latest"
+  image_password = "K3tTj8G14a3EgKyNeeiY"
+  type = "ENTERPRISE"
+  volume {
+    name = "system"
+    size = 5
+    disk_type = "SSD Standard"
+    user_data = "foo"
+    bus = "VIRTIO"
+    availability_zone = "ZONE_1"
+}
+  nic {
+    lan = ` + LanResource + `.` + LanTestResource + `.id
+    name = "system"
+    dhcp = true
+    firewall_active = true
+	firewall_type = "BIDIRECTIONAL"
```

Comment:
indentation

Reply:
@cristiGuranIonos don't know why it showed like this in `git`, in the file this line was looking all right. Anyway, I removed this since I removed the extraneous resources in a587496c4cd2e7e39ddca5d28621da37a154f51d.